.github/workflows/build-release.yml

```diff
@@ -0,0 +1,29 @@
+name: Build Release
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup proper PHP version
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+
+      - name: Install dependencies
+        run: composer install --no-dev --optimize-autoloader
+
+      - name: Zip the plugin
+        run: zip -r tumblr-importer.zip . -x ".*" -x "scripts/*"
+
+      - name: Upload plugin zip as release asset
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh release upload ${{ github.event.release.tag_name }} tumblr-theme-translator.zip
```

Comment:
The zip name here should be `tumblr-importer`.

So this work flow happens when a tag/release is created, right?

There is currently a deploy workflow which pushes to the WordPress.org SVN when a tag is made: https://github.com/WordPress/tumblr-importer/blob/trunk/.github/workflows/deploy.yml

I think that means the plugin directory would be getting the pre-built source. We probably need to update these workflows to work together.

Reply:
I moved this logic to the deploy file getting leverage of `generate-zip` from `10up/action-wordpress-plugin-deploy`